checks/checks.md

```diff
@@ -116,11 +104,23 @@ The check currently looks for [GitHub packaging workflows]( https://docs.github.
 - Publish your project as a [downloadable package](https://docs.github.com/en/packages/learn-github-packages/publishing-a-package).
 - Use a GitHub action to release your package to language-specific hubs.
 
+## Pinned-Dependencies 
+
+This check tries to determine if a project has declared and pinned its dependencies. 
+Pinning dependencies is important to mitigate compromised dependencies from undermining the security of the project. Low score is therefore considered `Medium` risk.
+The checks works by (1) looking for the following files in the root directory: go.mod, go.sum (Golang), package-lock.json, npm-shrinkwrap.json (Javascript),  requirements.txt, pipfile.lock (Python), gemfile.lock (Ruby), cargo.lock (Rust), yarn.lock (package manager), composer.lock (PHP), vendor/, third_party/, third-party/; (2) looks for  unpinned dependencies in Dockerfiles, shell scripts and GitHub workflows.  
+
+**Remediation steps**
+- Declare all your dependencies with specific versions in your package format file (e.g. `package.json` for npm, `requirements.txt` for python). For C/C++, check in the code from a trusted source and add a `README` on the specific version used (and the archive SHA hashes).
+- If the package manager supports lock files (e.g. `package-lock.json` for npm), make sure to check these in the source code as well. These files maintain signatures for the entire dependency tree and saves from future exploitation in case the package is compromised.
+- For Dockerfiles and github workflows, pin dependencies by hash. See example [gitcache-docker.yaml](https://github.com/ossf/scorecard/blob/main/.github/workflows/gitcache-docker.yaml#L36) and [Dockerfile](https://github.com/ossf/scorecard/blob/main/cron/worker/Dockerfile) examples.
+- To help update your dependencies after pinning them, use tools such as  Github's [dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/) or [renovate bot](https://github.com/renovatebot/renovate).
+
 ## Pull-Requests 
 
-This check tries to determine if the project requires pull requests for all changes to the default branch.
-Reviewing code improves quality of code in general. In addition, it ensures compromised contributors cannot intentionally inject malicious code. A low score is therefore considered `High` risk.
-The check works by looking at recent commits (first page, ~30) and uses the GitHub API to search for associated pull requests.  
+This check tries to determine if the project requires pull requests for  all changes to the default branch.
```

Comment:
remove the double spaces. can you search in file and remove them all.

Reply:
Done.